code/datums/components/orbiter.dm

```diff
@@ -0,0 +1,139 @@
+/datum/component/orbiter
```

Comment:
I'll be the first to admit, I'm not really a big fan of this component---it's turns a lightweight proc into something far more complicated.

That said, it does mean we can do things like `transfer_observers_to`, which does help cut down on thing, I suppose.

@Crazylemon64 would really like your opinion on all of this.